ui/src/main/webapp/components/MenuInput.stories.tsx

```diff
@@ -0,0 +1,126 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import React, { useState, useEffect } from 'react';
+import MenuInput from './MenuInput';
+import { setUnifiedConfig } from '../util/util';
+import { UnifiedConfig } from '../types/config';
+
+interface MenuInputProps {
+    handleRequestOpen: (args: { serviceName: string; input?: string; groupName?: string }) => void;
+    config: UnifiedConfig;
+}
+
+/*
+ * using wrapper to enable unified config passed in args/props
+ */
+function MenuInputWrapper(props: MenuInputProps) {
+    const [display, setDisplay] = useState(false);
+
+    useEffect(() => {
+        setUnifiedConfig(props.config);
+        setDisplay(true);
+    }, [props.config]);
+
+    return display ? (
+        <MenuInput handleRequestOpen={props.handleRequestOpen} />
```

Comment:
this is a story for controlled component. by showing the wrapper instead of pure component we may give a wrong perception that this component has state